### PR TITLE
Online-Anmeldung

### PR DESCRIPTION
**Pull request by lukasjuhrich**
WIP in pieces

### CODE REVIEW

File: sipa/blueprints/register.py [hunk 1]
```diff
@@ -0,0 +1,283 @@
...
+            elif e.code == 'no_room_for_tenancies':
+                reg_state.tenant_number = form.tenant_number.data
+                return goto_step('data')
+            else:
+                flash(gettext(
+                    'Die Verifizierung deiner Daten mit dem Studentenwerk Dresden ist fehlgeschlagen. '
+                    'Bitte überprüfe, dass du die exakt selben Daten, wie beim Studentenwerk Dresden '
+                    'bzw. wie auf deinem Mietvertrag, angegeben hast. '
+                    'Um die Verifizierung zu überspringen, kannst du den entsprechenden Button '
+                    'klicken. Die Verifizierung wird dann später manuell durchgeführt.'), 'error')
```
Comment (lukasjuhrich) [inline sipa/blueprints/register.py:161]: Quite a wall of text, the process is complicated enough anyway.
English suggestion:

> We could not match your data to a contract. Please check that it precisely matches the spelling in your rental agreement. Alternatively, you can choose to skip the automatic verification for now.

This is shorter and communicates everything necessary.

File: sipa/templates/register/identify.html [hunk 2]
```diff
@@ -0,0 +1,28 @@
+{% extends "register/base.html" %}#
+
+{% macro skip_btn() %}
+{% if skip_verification %}
+    <button type="submit" class="btn btn-danger" name="skip_verification">{{ _("Verifizierung überspringen") }}</button>
+{% endif %}
+{% endmacro %}
+
+{% block register_content %}
+    {{ forms.render_simple(form=form, form_id="register-form", submit_text=_("Weiter"), extra_btn=skip_btn) }}
```
Comment (lukasjuhrich) [inline sipa/templates/register/identify.html:10]: Situation-aware: If there was a mistake, label it `Continue`

Comment (lukasjuhrich) [reply sipa/templates/register/identify.html:10]: Also, the abort button could also be labeled „skip _automatic_ verification“ to make it more clear that it does not abort his application, it just delays the verification.

Comment (lukasjuhrich) [reply sipa/templates/register/identify.html:10]: CC @georgeto

**Review by ibot3 — APPROVED** [review]

**Review by georgeto — APPROVED** [review]

**Pull request merged**